cli/util/errors.go

```diff
@@ -0,0 +1,7 @@
+package util
+
+import "errors"
+
+var (
+	ErrCmdAbort = errors.New("aborted by user")
```

Comment:
Need comments for all external functions, vars, consts, types...

Reply:
Added comment for this error.